PerfDataExtension/README.md

```diff
@@ -0,0 +1,15 @@
+# Perf.data plugin

```

Comment:
Add missing pluginManifest.json for Perf Toolkit plugin system please similar to other projs